package.json

```diff
@@ -81,5 +55,34 @@
     "rimraf": "^2.5.2",
     "sinon": "^1.17.4",
     "sinon-chai": "^2.8.0"
-  }
+  },
+  "greenkeeper": {
+    "ignore": ["react", "react-dom"]
+  },
+  "homepage": "https://github.com/Travix-International/frint#readme",
+  "license": "MIT",
+  "main": "./lib/index.js",
+  "name": "frint",
```

Comment:
-.-' Of course we can, but IMHO, that just doesn't matter... Automated stuff couldn't care less and if you trust in the package.json's name attribute to know what's the project you're on, then ....

Still, reverting to non sorted.